WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Method/SimplePaymentsMethodsViewModel.swift

```diff
@@ -260,12 +269,22 @@ private extension SimplePaymentsMethodsViewModel {
     /// Observes the store CPP state and update publish variables accordingly.
     ///
     func bindStoreCPPState() {
-        cppStoreStateObserver
-            .statePublisher
-            .map { $0.isCompleted }
-            .removeDuplicates()
-            .assign(to: &$showPayWithCardRow)
-        cppStoreStateObserver.refresh()
+        ordersResultController.onDidChangeContent = updateCardPaymentVisibility
+        productResultsController.onDidChangeContent = updateCardPaymentVisibility
+        try? ordersResultController.performFetch()
+        try? productResultsController.performFetch()
+    }
+
+    func updateCardPaymentVisibility() {
+        print("💳 Updating visibility")
```

Comment:
Not a blocker, but perhaps we could use `DDLogDebug` or other log info instead of this `print`?